src/coreComponents/dataRepository/Group.hpp

```diff
@@ -1244,6 +1266,7 @@ class Group
 
   /**
    * @brief Return the path of this Group in the data repository.
+   * Starts with '/' followed up by the "Problem" children in which the Group is.
```

Comment:
Are you sure?

Reply:
I verified and I changed the comment to `Starts with '/' followed up by the hierarchy from the "Problem" children in which the Group is.`

Reply:
`Starts with '/' followed by the hierarchy of the children of the "Problem" in which the Group is.`?